app/forms/AssistanceDetailsForm.scala

```diff
@@ -60,7 +43,66 @@ object AssistanceDetailsForm {
                    needsSupportForOnlineAssessment: String,
                    needsSupportForOnlineAssessmentDescription: Option[String],
                    needsSupportAtVenue: String,
-                   needsSupportAtVenueDescription: Option[String]
-  )
+                   needsSupportAtVenueDescription: Option[String]) {
+
+    def exchange: AssistanceDetails = {
+      AssistanceDetails(
+        hasDisability,
+        hasDisabilityDescription,
+        guaranteedInterview.map {
```

Comment:
These pattern match unnecessary if we stick to string literals "true" or "false" instead of "Yes/No"